app.js

```diff
@@ -1,5 +1,13 @@
-// @ts-check
+//  _   _       _                       _          _    _       _       _               
+// | | | |     | |                     | |        | |  | |     | |     | |              
+// | | | | __ _| | ___  _ __ __ _ _ __ | |_ ______| |  | | __ _| |_ ___| |__   ___ _ __ 
+// | | | |/ _` | |/ _ \| '__/ _` | '_ \| __|______| |/\| |/ _` | __/ __| '_ \ / _ \ '__|
+// \ \_/ / (_| | | (_) | | | (_| | | | | |_       \  /\  / (_| | || (__| | | |  __/ |   
+//  \___/ \__,_|_|\___/|_|  \__,_|_| |_|\__|       \/  \/ \__,_|\__\___|_| |_|\___|_|   
+                                                                                     
```

Comment:
I don't think we need the banner, we already have a README.md

Reply:
Ok, you can remove it then.